cloud-sql-proxy/test_cloud_sql_proxy.py

```diff
@@ -56,9 +56,6 @@ def __submit_pyspark_job(self, cluster_name):
       'HA',
   )
   def test_cloud_sql_proxy(self, configuration):
```

Comment:
Seems like this test is failing during presubmit, may you fix it?

Reply:
Seems like HA test is still failing on Debian 10 images.

Reply:
I couldn't reproduce the error. Maybe flaky? I'm rerunning the test.

Reply:
Seems like it consistently fails in presubmit